frontend/packages/dev-console/src/components/monitoring/alerts/monitoring-alerts-utils.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { expandable, sortable, cellWidth } from '@patternfly/react-table';
+import { StateCounts, Severity, AlertState } from '@console/internal/components/monitoring';
+import { Kebab } from '@console/internal/components/utils';
+import {
+  alertDescription,
+  alertState,
+  AlertStates,
+  AlertSeverity,
+} from '@console/internal/reducers/monitoring';
+import { Alert, Rule } from '@console/internal/components/monitoring/types';
+
+const viewAlertRule = {
+  label: 'View Alerting Rule',
+};
+
+const alertStateReducer = (alert: Alert) => _.toLower(alert?.state);
```

Comment:
Shouldn't need to convert to lower case (same as https://github.com/openshift/console/pull/5785/files#r452628730)